### DISCUSSION TITLE
Playing, Streaming, Listening, Watching, Competing activity statuses

### DISCUSSION THREAD

**Discussion by KahpotVanilla** [maximmax42/Discord-CustomRP#173] [Ideas]
So, apparently there is a custom rich presence app on Android Open-sourced which actually offer this type of activities in their app, maybe you can make them into CustomRP too..
here is the github of it:
[https://github.com/dead8309/Kizzy](https://github.com/dead8309/Kizzy)

**Comment by maximmax42** [comment]
Yeah I'm aware of that app. It's using websockets, and I'm not smart enough to understand that xd. Maybe in the future, dunno.
